app/views/hyrax/base/_metadata.html.erb

```diff
@@ -1,8 +1,4 @@
-<h2><%= t('.header') %></h2>
-<table class="table table-striped <%= dom_class(presenter) %> attributes" <%= presenter.microdata_type_to_html %>>
-  <thead>
-    <tr><th><%= t('.attribute_name_label') %></th><th><%= t('.attribute_values_label') %></th></tr>
```

Comment:
remove these from the translation files?

Reply:
Done.